### PR TITLE
Fixed issues from SCCC meeting 4/10

### PR DESCRIPTION
**Pull request by YazidZidane**
- Fixed planting date warning error on Species Selection page.
- Refactored textfield with number input.
- Fixed SCCC percentOfRate calculation error, updated in Review Mix page.
- Updated Review Mix bar chart design.
- Updated styling in Seeding Method.
- Add adjustable single species seeding rate in Mix Ratios.
- Add Alert component noticing user can edit that step in further steps.
- Updated Seeding Method texts.

### CODE REVIEW

File: src/pages/Calculator/Steps/SeedingMethod/index.js [hunk 1]
```diff
@@ -246,55 +256,62 @@ const SeedingMethod = () => {
                 {council === 'MCCC'
                   && (
                   <>
-                    {renderMethod('Precision', methods[seed.label]?.Precision)}
-                    {renderMethod('Drilled', methods[seed.label]?.Drilled)}
+                    {renderMethod('Precision', methods[seed.label]?.Precision, '', 'Precision')}
```
Comment (mikahpinegar) [inline src/pages/Calculator/Steps/SeedingMethod/index.js:259]: why did you add another param with the same value? and why another param with empty string?

File: src/pages/Calculator/Steps/SeedingMethod/index.js [hunk 2]
```diff
@@ -246,55 +256,62 @@ const SeedingMethod = () => {
...
+                    {renderMethod('Drilled', methods[seed.label]?.Drilled, '', 'Drilled')}
```
Comment (mikahpinegar) [inline src/pages/Calculator/Steps/SeedingMethod/index.js:260]: why did you add another param with the same value? and why another param with empty string?

File: src/pages/Calculator/Steps/SeedingMethod/index.js [hunk 3]
```diff
@@ -246,55 +256,62 @@ const SeedingMethod = () => {
...
                     {renderMethod(
                       'Broadcast',
                       methods[seed.label]?.Broadcast,
                       'with Light Incorporation',
+                      'Broadcast',
```
Comment (mikahpinegar) [inline src/pages/Calculator/Steps/SeedingMethod/index.js:265]: why did you add another param with the same value?

File: src/pages/Calculator/Steps/SeedingMethod/index.js [hunk 4]
```diff
@@ -246,55 +256,62 @@ const SeedingMethod = () => {
...
                     )}
                     {renderMethod(
                       'Aerial',
                       methods[seed.label]?.Aerial,
                       'or broadcast with no Light Incorporation',
+                      'Aerial',
```
Comment (mikahpinegar) [inline src/pages/Calculator/Steps/SeedingMethod/index.js:271]: why did you add another param with the same value?

File: src/pages/Calculator/Steps/SeedingMethod/index.js [hunk 5]
```diff
@@ -246,55 +256,62 @@ const SeedingMethod = () => {
...
                     )}
                   </>
                   )}
                 {council === 'NECCC'
                   && (
                   <>
-                    {renderMethod('Drilled', methods[seed.label]?.Drilled)}
+                    {renderMethod('Drilled', methods[seed.label]?.Drilled, '', 'Drilled')}
```
Comment (mikahpinegar) [inline src/pages/Calculator/Steps/SeedingMethod/index.js:278]: why did you add another param with the same value?

File: src/pages/Calculator/Steps/SeedingMethod/index.js [hunk 6]
```diff
@@ -246,55 +256,62 @@ const SeedingMethod = () => {
...
                     {renderMethod(
                       'Broadcast',
                       methods[seed.label]?.['Broadcast(With Cultivation)'],
                       'with Cultivation, No Packing',
+                      'Broadcast(With Cultivation)',
```
Comment (mikahpinegar) [inline src/pages/Calculator/Steps/SeedingMethod/index.js:283]: why did you add another param with the same value?

File: src/pages/Calculator/Steps/SeedingMethod/index.js [hunk 7]
```diff
@@ -246,55 +256,62 @@ const SeedingMethod = () => {
...
                     )}
                     {renderMethod(
                       'Broadcast',
                       methods[seed.label]?.['Broadcast(With No Cultivation)'],
                       'with No Cultivation, No Packing',
+                      'Broadcast(With No Cultivation)',
                     )}
-                    {renderMethod('Aerial', methods[seed.label]?.Aerial)}
+                    {renderMethod('Aerial', methods[seed.label]?.Aerial, '', 'Aerial')}
```
Comment (mikahpinegar) [inline src/pages/Calculator/Steps/SeedingMethod/index.js:291]: why did you add another param with the same value?

File: src/pages/Calculator/Steps/SeedingMethod/index.js [hunk 8]
```diff
@@ -246,55 +256,62 @@ const SeedingMethod = () => {
...
                   </>
                   )}
                 {council === 'SCCC'
                   && (
                   <>
-                    {renderMethod('Drilled', methods[seed.label]?.Drilled)}
+                    {renderMethod('Drilled', methods[seed.label]?.Drilled, '', 'Drilled')}
```
Comment (mikahpinegar) [inline src/pages/Calculator/Steps/SeedingMethod/index.js:297]: why did you add another param with the same value?

**Review by mikahpinegar — CHANGES_REQUESTED** [review]

File: src/pages/Calculator/Steps/SeedingMethod/index.js (hunk 1 above)
Comment (YazidZidane) [reply src/pages/Calculator/Steps/SeedingMethod/index.js:259]: In Seeding Method page, The first param is the Bolded seeding method at left(`Precision`, `Drilled`, etc), the second param is the value at right, the third param is the text under the seeding method(`with Light Incorporation`, etc), the fourth param is the selected seeding method(the method in dropdown).

This function do contains many duplicate params but it needs to handle different situations in councils, I can reduce the length of the param by reusing the method name in second param.

**Review by mikahpinegar — APPROVED** [review]
